### ISSUE TITLE
Pointer returns a far-away gameobject to be Near when its scale is Zero

### ISSUE THREAD

**Issue by JoshuaChoiSynchro** [microsoft/MixedRealityToolkit-Unity#5656]
## Describe the bug

When a gameobject has a scale of (0,0,0), the hand pointer returns it as a Near object even if the gameobject is far away.

Potential issue at NearInteractionTouchable.cs at line 214.
Vector3 localPoint = transform.InverseTransformPoint(samplePoint) - localCenter;

## To reproduce

1. Create primitive Box (in MRTK ready scene)
2. attach NearInteractionTouchable.cs
3. scale the Box to 0,0,0
4. PokePointer will always return it as Near object when the Box is far away.

## Expected behavior

Should not return as a Near object when the object is far away even when scale is zero.

## Screenshots

N/A

## Your Setup (please complete the following information)

- Unity Version 2019.1.14f
- MRTK Version 2.0.0 RC2.1

## Target Platform (please complete the following information)

- Editor
- HoloLens 2

**Comment by CDiaz-MS** [comment]
The issue has been resolved in the latest release MRTK 2.0.0.  If you see this on a future release please open again.

**Comment by CDiaz-MS** [comment]
Checking if the poke pointer is near an object:

```
        foreach (var source in CoreServices.InputSystem.DetectedInputSources)
        {
            if (source.SourceType == InputSourceType.Hand)
            {
                foreach (var p in source.Pointers)
                {
                    if (p is PokePointer)
                    {
                        var poke = p as PokePointer;
                        UnityEngine.Debug.Log("Poke Pointer isNearObject: " + poke.IsNearObject);
                    }
                }
            }
        }

```
